syntax = "proto3";

//python -m grpc_tools.protoc   -I.. --python_out=. --grpc_python_out=.   ../msgs_services.proto   (for generating the python files)

service Server {
  rpc GetTransactionData (TransactionRequest) returns (TransactionReply) {}
}

message TransactionRequest {
  string name = 1;
}

message TransactionReply {
  string meas_info = 1;
  string counter = 2;
  string cell_name = 3;
  string value = 4;
}
